### PR TITLE
Fix multiple spelling mistakes

### PR DESCRIPTION
**Pull request by ggoncalopereira**
Attempt to fix a bunch of spelling mistakes.
Couldn't test locally for any discrepancies (HTML ellipsis, apostrophes, quotes...) but I'll have to trust my gut on this one.

### CODE REVIEW

**Comment by ggoncalopereira** [comment]
Also had to guess the meaning of some sentences, but hopefully the main essence of it is still present.

**Review by nelsonmestevao — CHANGES_REQUESTED** [review]
We don't edit the `HTML` files directly. They are automatically generated from [hugo](https://gohugo.io/).

The source code for the website tutorial is found under [`src/`](https://github.com/cesium/fractal-tree/tree/master/src) and I believe the file you are looking for is [`_index.md`](https://github.com/cesium/fractal-tree/blob/master/src/content/_index.md).

**Comment by nelsonmestevao** [comment]
@ggoncalopereira can you drop the first commit? :pray:

**Comment by ggoncalopereira** [comment]
@nelsonmestevao my bad - updated the .md and only pushed the .HTML commit. Should be fixed soon

File: src/content/_index.md [hunk 1]
```diff
@@ -6,20 +6,22 @@ weight: -270
...
+explain what a fractal is. To begin with, a fractal is the infinite repetition of a
+certain pattern - in this particular case our pattern is the "Y" symbol. This
+concept may not seem clear for now but don't worry - it will be soon! If you have trouble with
 fractals you can find more about it [`here`](https://mathigon.org/world/Fractals)
 before starting this tutorial.
 
-Now that you know what a fractal is, a fractal tree is simply a fractal that builds
-a tree as it is being created. Okay, after this basic explanation lets get it started!
+Now that you know what a fractal is, a fractal tree is simply a fractal that builds a
+tree as it is being created. Okay, after this basic explanation let&rsquo;s get it
```
Comment (nelsonmestevao) [inline src/content/_index.md]: Use `'` here, please :pray:

File: src/content/_index.md [hunk 2]
```diff
@@ -94,9 +97,9 @@ In this example the canvas has a 200x200 dimension.
 
 #### Background
 
-Since a blank canvas is boring and programming is not, you need to give it a
-little bit of life. To do so, you use a background that allows editing your
-canvas color from 0 to 255, 0 being black and 256 white, respectively. Read
+Since a blank canvas is boring and programming is not, you need to give it a
```
Comment (nelsonmestevao) [inline src/content/_index.md]: Remove the extra space (end of `a`) :pray:

File: src/content/_index.md [hunk 3]
```diff
@@ -111,14 +114,13 @@ The background is now Black.
...
 
-With your "base" now set, lets learn how to draw. In this case we will be
-drawing lines, so what do you need to draw a line? Simple answer: exactly two
-points. So give the function two points (height, width) and let `line` do its
-job, unite those points. See? As easy as it can be. Futher `line` knowledge 
-[`here`](https://p5js.org/reference/#/p5/line)
+With your "base" now set, let's learn how to draw. In this case we will be
+drawing lines - and what do you need to draw a line? Simple answer: exactly two
+points, no more, no less. So give the function two points (height, width) and let the `line` do its
+job by connecting those points. See? As easy as it can be. See more [`here`](https://p5js.org/reference/#/p5/line)
```
Comment (nelsonmestevao) [inline src/content/_index.md]: Please, try to keep the text under 80 characters :point_up:

**Comment by ggoncalopereira** [comment]
LGTM, should hopefully be good to go @nelsonmestevao

File: src/content/_index.md [hunk 4]
```diff
@@ -445,15 +444,15 @@ It still doesn't work.
...
+Why did it work so well? Remember the `translate` function. Since you
 always changed the "starting" point to the top of the branch it doesn't come
-back to make it to the left side. Instead it goes right and builds the
-pattern only there making, a funny but not intended result. To solve this
-problem is quite easy. Use push and pop, push saves the initial position that
-you were in, before applying the if condition then you do branch like usually and
-when the if condition makes you stop you pop, it means you go back to the state
-saved in push. Up to this point you have only done the right side of the tree
-now for the left side just push and pop again but this time to the negative
+ck to interfere with the left side. Instead it goes right and builds the
```
Comment (nelsonmestevao) [inline src/content/_index.md]: `ck` should be `back`

**Review by nelsonmestevao — CHANGES_REQUESTED** [review]

**Comment by ggoncalopereira** [comment]
Latest commit should fix all those issues - also pushed back previous wrongly indented sentences to keep the margin of 80 characters, max.

**Comment by ggoncalopereira** [comment]
Actually - one second, found a bunch more things...

**Comment by ggoncalopereira** [comment]
Should be good now!

**Comment by ggoncalopereira** [comment]
Should be good now!

**Comment by ggoncalopereira** [comment]
@nelsonmestevao any updates on this? Pinging to ask, that way the repo can be deployed and/or used whenever

**Comment by nelsonmestevao** [comment]
I am coming back to this asap :muscle:

**Pull request merged**